### PR TITLE
Fix default extinction correction

### PR DESCRIPTION
**Pull request by finagle29**
Extinction correction should be applied by default if the sensfunc algorithm is UVIS, this PR fixes a typo to make this true.

### CODE REVIEW

File: doc/pypeit_par.rst [hunk 1]
```diff
@@ -531,12 +531,12 @@ FluxCalibratePar Keywords
...
+``extinct_correct``  bool  ..       ..       The default behavior for atmospheric extinction corrections is that if UVIS algorithm is used (which does not correct for telluric absorption) than an atmospheric extinction model is used to correct for extinction below 10000A, whereas if the IR algorithm is used, then no extinction correction is applied since the atmosphere is modeled directly. To follow thesedefaults based on the algorithm this parameter should be set to extinct_correct=None. If instead this parameter is set, this overide this default behavior. In other words, it will force an extinction correctionif extinct_correct=True, and will not perform an extinction correction if extinct_correct=False.
```
Comment (rcooke-ast) [inline doc/pypeit_par.rst:538]: `thesedefaults` => `these defaults`
`correctionif` => `correction if`

**Review by rcooke-ast — APPROVED** [review]
Seems simple enough!

**Review by jhennawi — APPROVED** [review]
Thanks for catching this!

**Comment by finagle29** [comment]
This didn't actually fix the issue, I don't think the algorithm is being read from sens.fits files.

**Pull request merged**